### PR TITLE
feat(HMS-1879): Add instance desc table for GCP

### PR DESCRIPTION
**Pull request by adiabramovitch**
This PR adds an instance description table at the end of a successful launch and also adds missing tests. 
image

### CODE REVIEW

**Comment by lzap** [comment]
Shall we remove the DNS at some point? I mean, only AWS supports public DNS by default, maybe showing up SSH is enough.

Other than that, looks good.

**Comment by adiabramovitch** [comment]
> Shall we remove the DNS at some point? I mean, only AWS supports public DNS by default, maybe showing up SSH is enough.
> 
> 
> 
> Other than that, looks good.

In a followup PR we can show the DNS column only if the DNS(s) exists.

File: src/Components/ProvisioningWizard/steps/SourceMissing/SourceMissing.test.js [hunk 1]
```diff
@@ -70,4 +70,28 @@ describe('Source missing', () => {
...
+      render(<SourceMissing image={image} />);
+
+      const sourcesLink = screen.getByRole('link', { name: 'Create Source' });
+      expect(sourcesLink).toHaveAttribute('href', '/preview/settings/sources/new');
+    });
+
+    test('renders direct link', () => {
+      render(<SourceMissing image={image} />);
+
+      const url = 'https://console.cloud.google.com/welcome';
```
Comment (ezr-ondrej) [inline src/Components/ProvisioningWizard/steps/SourceMissing/SourceMissing.test.js:91]: Do we have a ticket for this? I've noticed this for second time, but I guess we do not want to fix this now.
What I mean is opening a welcome page is not very meaningful prolly :) 
We shall have a ticket to remember tho :)

**Review by ezr-ondrej — COMMENTED** [review]
Looks good, just verifying we've got a ticket for the welcome redirect, apart of it looks good :+1:

**Comment by adiabramovitch** [comment]
> Looks good, just verifying we've got a ticket for the welcome redirect, apart of it looks good :+1: 

We have a ticket for it, it is in the backlog :)

**Review by ezr-ondrej — APPROVED** [review]
Thanks! :+1:

**Pull request merged**